baseplate/random.py

```diff
@@ -39,26 +40,26 @@ class WeightedLottery(object):
     """
 
     def __init__(self, items, weight_key):
-        self.tickets = 0
-        self.items_with_weights = []
+        self.weights = []
+        self.items = list(items)
```

Comment:
More nerd sniping happening: I did some brief reading on this, and the possibilities that come to mind are:

1. `tuple` construction is generally faster than `list` construction and accessing tuples is sometimes faster as well (http://stackoverflow.com/questions/3340539/why-is-tuple-faster-than-list) 
2. may be slightly more space efficient as lists over-allocate memory IIRC
3. since we're only iterating through the data (no sorting, mutation) there's no functionality loss.